sources/packages/web/src/components/common/ManageProgramAndOfferingSummary.vue

```diff
@@ -11,6 +11,7 @@
       <offering-summary
         :programId="programId"
         :locationId="locationId"
+        :canCreateOffering="educationProgram.isActive"
```

Comment:
As the program id is sent to the component, isActive status of program can be derived inside the component.